orca-core/src/main/java/com/netflix/spinnaker/orca/config/OrcaConfiguration.java

```diff
@@ -75,7 +75,8 @@
   "com.netflix.spinnaker.orca.pipeline.util",
   "com.netflix.spinnaker.orca.preprocessors",
   "com.netflix.spinnaker.orca.telemetry",
-  "com.netflix.spinnaker.orca.notifications.scheduling"
+  "com.netflix.spinnaker.orca.notifications.scheduling",
+  "com.netflix.spinnaker.plugin"
```

Comment:
What's the purpose of this?

Reply:
To be able to component scan plugins. Right now if you create a project including `orca-api` and implement StageDefintionBuilder, it will be component scanned if you drop the resulting jar into `/opt/spinnaker/plugins`. This is just a simple way to test plugins for right now

Reply:
I think the intended scope of this PR is to pull out the building blocks that can be used by plugin developers. It would probably be best to exclude the 'loader' code, especially since it is really just in there for temporary testing.